scripts/observers/config.py

```diff
@@ -0,0 +1,113 @@
+import os
+import toml
+import sys
+import shutil
+from pathlib import Path
+from git import Repo
```

Comment:
This does not seem from the standard library. Perhaps add a `requirements.txt` file and a README explanation?

For example:
 - https://github.com/ElrondNetwork/db-archive-scripts/blob/main/requirements.txt
 - https://github.com/ElrondNetwork/db-archive-scripts/blob/main/README.md#prerequisites

Reply:
done